app/src/androidTest/java/no/ntnu/klubbhuset/ui/userviews/club/ClubsViewModelTest.java

```diff
@@ -0,0 +1,24 @@
+package no.ntnu.klubbhuset.ui.userviews.club;
```

Comment:
If this is a testclass, maybe it should be placed in the test package?

"IE303911-Klubbhuset/app/src/test/java/no/ntnu/klubbhuset"

Reply:
Sjå igjen, vegen til klassa er "app/src/androidTest/java/no/ntnu/klubbhuset/ui/userviews/club/ClubsViewModelTest.java "